infrastructure/main.tf

```diff
@@ -3,44 +3,6 @@ data "azurerm_key_vault" "sscs_key_vault" {
   resource_group_name = "${local.vaultName}"
 }
 
-data "azurerm_key_vault_secret" "cookiesecret" {
-  name      = "tyacookiesecret"
```

Comment:
Do we need to remove this? is it not being used?

Reply:
I think it is safe to do this as it's getting the value from AKS config.